meeting_minutes/20220930_SARIF_TC_61.md

```diff
@@ -0,0 +1,153 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair David)
+
+## 1.2 Introduction of participants/roll call (Co-Chair David)
+
+| First Name | Last Name | Company           | Role(s)                 |
+|:-----------|:----------|:------------------|:------------------------|
+| Chris      | Meyer     | Microsoft         | Voting Member           |
+| Chris      | Wysopal   | Veracode          | Member                  |
+| David      | Keaton    | Individual        | Chair                   |
+| Jeff       | Williams  | Contrast Security | Voting Member           |
+| Mary       | Martin    | Microsoft         | Voting Member           |
+| Nathan     | Baird     | Microsoft         | Voting Member           |
+| Pavel      | Mikula    | Sonar             | Member                  |
+| Thanassis  | Avgerinos | ForAllSecure Inc  | Voting Member           |
+| Yekaterina | O'Neil    | Micro Focus       | Voting member           |
+
+## 1.3 Procedures for this meeting (Co-Chair David)
+
+## 1.4 Approval of agenda (Co-Chair David)
+
+* https://www.oasis-open.org/committees/download.php/70111/agenda_20220630.html
+
+Agenda was approved.
+
+## 1.5 Approval of previous minutes (Co-Chair David)
+
+* [Minutes of 2022-06-16_Meeting #60](https://www.oasis-open.org/committees/document.php?document_id=70061&wg_abbrev=sarif)
+
+Minutes were approved.
+
+## 1.6 Review of action items and resolutions (Co-Chair David)
+
+* ACTION on Michael to initiate contact with OpenSSF
+  * ONGOING - in progress, but delayed because of people on vacation
+* ACTION on Chris M. to solicit a working example from WhiteSource of their scenario
+  * ONGOING - still ongoing but the demo was already shared (tool now called Mend)
+ 
+## 1.7 Identification of SARIF TC voting members (Co-Chair David)
+
+### 1.7.1 Prospective members attending their first meeting
+
+* Chris Wysopal will obtain voting rights at the end of this meeting.
+* Pavel Mikula will obtain voting rights at the end of this meeting.
```

Comment:
Should these lines be under 1.7.2?

Reply:
I think they should, it was 2nd meeting in a row for both of us.